src/databases/Mili/avtMiliFileFormat.C

```diff
@@ -768,7 +768,7 @@ avtMiliFileFormat::GetMesh(int timestep, int dom, const char *mesh)
                 // Create a copy of our name to pass into mili. 
                 //
                 char charName[1024];
-                sprintf(charName, varName.c_str());
+                sprintf(charName, "%s", varName.c_str());
```

Comment:
Here and elsewhere, we prefer you use snprintf. Because it is _snprintf on Windows,  there is a macro (SNPRINTF) defined in src/common/utility/snprintf.h that #defines the correct name for the platform.

Reply:
Done